app/javascript/mastodon/components/status_content.js

```diff
@@ -68,11 +70,11 @@ class StatusContent extends React.PureComponent {
     const [ startX, startY ] = this.startXY;
     const [ deltaX, deltaY ] = [Math.abs(e.clientX - startX), Math.abs(e.clientY - startY)];
 
-    if (e.target.localName === 'button' || e.target.localName === 'a' || (e.target.parentNode && e.target.parentNode.localName === 'a')) {
+    if (e.target.localName === 'a' || (e.target.parentNode && e.target.parentNode.localName === 'a')) {
```

Comment:
are you sure? cause we introduced a regression where clicking the CW button would also open the detailed status and after this changeset it is fixed, which I suspect is related to this  particular change though i didn't test it in isolation